src/sentry/integrations/slack/actions/notification.py

```diff
@@ -162,106 +154,70 @@ def send_notification(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
                     )
                     # To reply to a thread, use the specific key in the payload as referenced by the docs
                     # https://api.slack.com/methods/chat.postMessage#arg_thread_ts
-                    payload["thread_ts"] = parent_notification_message.message_identifier
                     thread_ts = parent_notification_message.message_identifier
                     # If this flow is triggered again for the same issue, we want it to be seen in the main channel
-                    payload["reply_broadcast"] = True
                     reply_broadcast = True
 
-            organization = event.group.project.organization
+            client = SlackSdkClient(integration_id=integration.id)
+            text = str(blocks.get("text"))
+            try:
+                response = client.chat_postMessage(
+                    blocks=json_blocks,
+                    text=text,
+                    channel=channel,
+                    unfurl_links=False,
+                    unfurl_media=False,
+                    thread_ts=thread_ts,
+                    reply_broadcast=reply_broadcast,
+                )
+                metrics.incr(
+                    SLACK_ISSUE_ALERT_SUCCESS_DATADOG_METRIC,
+                    sample_rate=1.0,
+                    tags={
+                        "action": "send_notification",
+                        "ok": response.get("ok", False),
+                        "status": response.status_code,
```

Comment:
maybe i don't need to be logging these for success?

Reply:
does the status code matter for us on success?

Reply:
no :P